service/service_test.go

```diff
@@ -128,9 +128,10 @@ func TestNewCaptionsService(t *testing.T) {
 	assert.NotNil(service.client.Providers)
 	assert.Equal(service.client.Providers, providers)
 
-	assert.Len(service.Endpoints(), 4)
+	assert.Len(service.Endpoints(), 5)
```

Comment:
I'd delete this line, I don't think the test should care about how many endpoints the service has, only which endpoints x)